src/styles/blocks/catalog.scss

```
@@ -18,6 +20,14 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition-duration: 300ms;
+    transition-timing-function: linear;
```

don't use it like that, because it's the same as transition: all 0.3s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md

```suggestion
    transition: transform $transitionDuration, color $transitionDuration
```

---

Also, create variable for transition duration